proto/xudrpc.proto

```diff
@@ -423,6 +463,25 @@ message Channels {
   uint32 closed = 4 [json_name = "closed"];
 }
 
+message ChannelBalanceAlert {
+  // The total balance of the channel.
+  uint64 total_balance = 1 [json_name = "total_balance"];
+  // The side of the low balance.
+  enum Side {
```

Comment:
Since this is used in two places we can move it out so it's not nested within another message and only define it once. I also think `ChannelSide` is more descriptive.